### PR TITLE
feat(route): 支持订阅「世界黄金协会」网站中「新闻与活动」版块的文章

### PR DESCRIPTION
**Pull request by BiosSun**
## Involved Issue / 该 PR 相关 Issue

无

## Example for the Proposed Route(s) / 路由地址示例

```routes
/gold/news
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Anti-bot or rate limit / 反爬/频率限制
  - [x] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

支持订阅「世界黄金协会」网站中「新闻与活动」版块的文章。

### CODE REVIEW

**Comment by BiosSun** [comment]
> Didn't pass auto route test. log: https://github.com/DIYgod/RSSHub/actions/runs/8958688165/job/24603205767#step:13:19

Oh, The route in PR is wrong, now fixed.

File: lib/routes/gold/news.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import { load } from 'cheerio'; // 类似 jQuery 的 API HTML 解析器
+import cache from '@/utils/cache';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/news',
+    name: '新闻与活动',
+    maintainers: ['BiosSun'],
+    example: '/news',
```
Comment (TonyRL) [inline lib/routes/gold/news.ts:12]:
```suggestion
    example: '/gold/news',
```

File: lib/routes/gold/news.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+        const items = await Promise.all(
+            list.map(
+                (item) =>
+                    cache.tryGet(item.link, async () => {
+                        const response = await ofetch(item.link);
+                        const $ = load(response);
+                        return {
+                            ...item,
+                            description: $('main section.field-main-content').html() || $('main .wgc-text').html() || '',
```
Comment (TonyRL) [inline lib/routes/gold/news.ts:53]: reassign the value of `descrition` and avoid spreading `item`

File: lib/routes/gold/news.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                        };
+                    }) as Promise<typeof item>
+            )
+        );
+
+        return {
+            // 源标题
+            title: `World Gold Council News & Events`,
```
Comment (TonyRL) [inline lib/routes/gold/news.ts:61]:
```suggestion
            title: 'World Gold Council News & Events',
```

File: lib/routes/gold/namespace.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import type { Namespace } from '@/types';
+
+export const namespace: Namespace = {
+    name: '世界黄金协会',
+    url: 'china.gold.org',
```
Comment (TonyRL) [inline lib/routes/gold/namespace.ts:5]:
```suggestion
    url: 'www.gold.org',
```

**Pull request closed without merging**